Answer.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Gives answers.
+ */
+public class Answer {
+  /**
+   * Prints an answer to the console to the task of the program (@link Order).
+   * @param analyzed is flag to give one of two answer.
+   */
+  public void giveAnswer(boolean analyzed) {
```

Comment:
non-appropriate argument name (it says only whether sequence has been analyzed or not)